npm/react/examples/a11y/yarn.lock

```diff
@@ -808,7 +808,7 @@
     nyc "15.1.0"
 
 "@cypress/react@file:../../dist":
-  version "0.0.0-development"
+  version "0.0.0-developmentelopment"
```

Comment:
I am sure you noticed this one already but in case you didnt @panzarino spelling error

Reply:
yup - was doing find all & replace and accidentally started modifying the yarn.lock, regenerated lockfile coming when I push